packages/pluggableWidgets/bar-chart-web/src/BarChart.editorPreview.tsx

```diff
@@ -0,0 +1,26 @@
+import { createElement, ReactNode } from "react";
+import { ChartWidget } from "@mendix/shared-charts";
+
+export function preview(): ReactNode {
+    // TODO:
```

Comment:
Once Diego asked me to remove todo's from files.
But I believe it's better to have it if there is a context inside that, but in this case I guess it can be removed

Reply:
This one is needed as "WIP" indicator, I think